index.html

```diff
@@ -5,8 +5,8 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="styles.css">
-  <script src="data.js"></script>
-  <script src="index.js"></script>
+  <script src="data.js" defer></script>
+  <script src="index.js" defer></script>
```

Comment:
Nice job! You actually don't need to defer the first script tag but it's ok that you did.